bazel/grpc_deps.bzl

```diff
@@ -519,3 +519,14 @@ def grpc_test_only_deps():
             ],
             build_file = "@com_github_grpc_grpc//third_party:constantly.BUILD",
         )
+
+    if "com_google_libprotobuf_mutator" not in native.existing_rules():
+        http_archive(
+            name = "com_google_libprotobuf_mutator",
+            sha256 = "b847c71723d8ce0b747aa661d7f3a07f1d16c595bf9c0202f30febc2f9a24a06",
+            urls = [
```

Comment:
You'll want to add [a mirror](https://github.com/grpc/grpc/blob/dfae8de8638e25dd212d1783a7cc67a1dc1fc485/bazel/grpc_deps.bzl#L195) as well to  protect us from GitHub outages. [This script](https://github.com/grpc/grpc/blob/dfae8de8638e25dd212d1783a7cc67a1dc1fc485/bazel/update_mirror.sh) will help you with that.